hazelcast/src/test/java/com/hazelcast/internal/partition/impl/InternalMigrationListenerTest.java

```diff
@@ -27,7 +28,7 @@
 
     private static final int PARTITION_COUNT = 2;
 
-    @Test
+    @Test @Ignore
```

Comment:
@metanet  will take care of it. It should be unrelated from the implementations here.